docs/RadioGroup.md

```diff
@@ -0,0 +1,34 @@
+### RadioGroup
+
+ **Props**
+
+| Property | Type | Required? | Default | Description |
```

Comment:
add true false  values in the required column

Reply:
We are using this format for the props table

| Property | Type | Required? | Default | Description |
|:---|:---:|:---:|:---:|:---|
| className | String | | `''` | Some style |
| value | Number | ✓ | | Some value |